gunicorn/arbiter.py

```diff
@@ -340,9 +340,12 @@ def wakeup(self):
     def halt(self, reason=None, exit_status=0):
         """ halt arbiter """
         self.stop()
-        self.log.info("Shutting down: %s", self.master_name)
+
+        log_func = self.log.info if exit_status == 0 else self.log.error
+        log_func("Shutting down: %s", self.master_name)
```

Comment:
why not simply adding the log error as a second line ? I think displaying "shutting down" in info logs is useful. (sorry to not have take that earlier).